### PR TITLE
Feature/test tests

### PR DESCRIPTION
**Pull request by lexenington**
Fixed all failing tests

### CODE REVIEW

**Review by utahguy — APPROVED** [review]

File: subscriptions/admin.py [hunk 1]
```diff
@@ -5,16 +5,42 @@
 from subscriptions.conf import SETTINGS
 
 
-class PlanCostInline(admin.TabularInline):
-    """Inline admin class for the PlanCost model."""
-    model = models.PlanCost
+# class PlanCostInline(admin.TabularInline):
```
Comment (AndjDavis) [inline subscriptions/admin.py]: Could we get a note explaining the commented out code? Do we still want it in the project?  Any future plans with it?

File: subscriptions/admin.py [hunk 2]
```diff
@@ -5,16 +5,42 @@
...
         'cost',
     )
-    extra = 0
+    list_display = (
+        'slug',
+        'recurrence_period',
+        'recurrence_unit',
+        'cost',
+    )
+    # prepopulated_fields = {'slug': ('plan_name',)}
```
Comment (AndjDavis) [inline subscriptions/admin.py]: Is prepopulated_fields ever going to be used again?

File: subscriptions/views.py [hunk 3]
```diff
@@ -250,10 +250,17 @@ def get(self, request, *args, **kwargs):
     def post(self, request, *args, **kwargs):
         """Overriding post method to handle inline formsets."""
         # Setup the formset for PlanCost
-        PlanCostFormSet = inlineformset_factory(  # pylint: disable=invalid-name
+        # PlanCostFormSet = inlineformset_factory(  # pylint: disable=invalid-name
```
Comment (AndjDavis) [inline subscriptions/views.py]: can we get rid of this commented out code?

File: tests/subscriptions/test_views_plans.py [hunk 4]
```diff
@@ -21,14 +25,17 @@ def create_plan(plan_name='1', plan_description='2'):
     )
 
 
-def create_plan_cost(plan, rec_period=1, rec_unit=models.MONTH, cost='1.00'):
+def create_plan_cost(plan=None, rec_period=1, rec_unit=models.MONTH, cost='1.00'):
     """Creates and returns a PlanCost instance."""
-    return models.PlanCost.objects.create(
-        plan=plan,
+    pc = models.PlanCost.objects.create(
+        # plan=plan,
```
Comment (AndjDavis) [inline tests/subscriptions/test_views_plans.py]: we keeping plan=plan?

File: tests/subscriptions/test_views_plans.py [hunk 5]
```diff
@@ -608,6 +662,7 @@ def test_plan_update_invalid_form(admin_client):
 @pytest.mark.django_db
 def test_plan_update_invalid_cost_forms(admin_client):
     """Tests handling of invalid costforms submission during update."""
+    # TODO: rename function create_subscription_plan
```
Comment (AndjDavis) [inline tests/subscriptions/test_views_plans.py]: We still need this todo here?

**Review by AndjDavis — APPROVED** [review]
Some left over commented out code, some todos left in there. This is a monster PR.  Good work!

**Pull request merged**